import { translation as _ } from '@atb/translations/commons';

export const Error = {
  title: _(
    'Oisann! Noe har gått galt...',
    'Oh no, something is wrong...',
    'Oisann! Noko har gått gale...',
  ),
  backButton: _('Tilbake', 'Back', 'Tilbake'),
  '404': _(
    'Fant ikke siden du prøvde å åpne. Forsøk å gå tilbake til startsiden og prøv på nytt.',
    "Couldn't find the page you were looking for. Please try returning to the main page and try again.",
    'Fann ikkje sida du forsøkte å gå til. Du kan forsøke å gå til startsida og forsøke ein gang til.',
  ),
  '500': _(
    'Her gikk vi på en blemme og det skjedde en feil. Kan du prøve igjen?',
    'An error occured. Could you try again for us?',
    'Her gjekk vi på ei blemme og det skjedde ein feil. Kan du prøve igjen?',
  ),
  unknown: _(
    'Her gikk vi på en blemme og det skjedde en feil. Kan du prøve igjen?',
    'An error occured. Could you try again for us?',
    'Her gjekk vi på ei blemme og det skjedde ein feil. Kan du prøve igjen?',
  ),
};
